test/api/extract.test.js

```diff
@@ -29,7 +29,7 @@ assert.strictEqual(
 );
 assert.strictEqual(
   map.scopes["https://ga.jspm.io/"]["object-assign"],
-  "https://ga.jspm.io/npm:object-assign@4.1.0/index.js"
+  "https://ga.jspm.io/npm:object-assign@4.1.1/index.js"
```

Comment:
A lot of these test changes are from secondaries being bumped to latest compatible.

Reply:
Could we add `freeze` to the test case? That will also help avoid new releases shifting our tests which is a common issue.